tests/SbrpTests/ValidationTests.cs

```diff
@@ -0,0 +1,178 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging;
+using NuGet.Packaging.Signing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SbrpTests;
+
+public class ValidationTests
+{
+    private const string SbrpAttributeType = "System.Reflection.AssemblyMetadataAttribute";
+    private const string VersionPattern = @"(\.\d)+([\-\w])*";
+    public ITestOutputHelper Output { get; set; }
+
+    public ValidationTests(ITestOutputHelper output)
+    {
+        Output = output;
+        
+        if (string.IsNullOrWhiteSpace(Config.RepoRoot))
```

Comment:
Ideally this logic wouldn't have to be duplicated across the tests.  Consider encapsulating it in the Config class.  Ideally getters wouldn't [throw](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/property) but this isn't a framework so I think in this case in-lining the validation in the getter is acceptable.

Reply:
The main issue is that the environment variable could still be set to whitespace or empty, so it would not evaluate to the RHS of the null-coalescing operator in this instance.

I could do something such as:

```
public static string RepoRoot
    {
        get
        {
            string repoRoot = Environment.GetEnvironmentVariable(RepoRootEnv) ??
                Environment.CurrentDirectory.Substring(0, Environment.CurrentDirectory.IndexOf("artifacts"));
            if (string.IsNullOrWhiteSpace(repoRoot))
            {
                throw new InvalidOperationException($"Environment variable {RepoRootEnv} cannot be null, empty, or whitespace.");
            }
            return repoRoot;
        }
    }
```

In some of the vmr smoke tests, we follow the pattern of checking if the string is null, empty, or whitespace in the test itself.

Reply:
Alternatively, I could also rely on letting the test itself fail with a directory does not exist exception.

Reply:
Addressed in https://github.com/dotnet/source-build-reference-packages/pull/817/commits/417c9953c921b03eecb143ae7fb4dec5a5fc4fcc

Reply:
Given this validation was moved to the config.cs file, is the check/throw necessary here anymore?